infrastructure/stacks/nhse-mng-uec-dos-domain-s3-artefacts/s3.tf

```diff
@@ -0,0 +1,46 @@
+module "sm_artefacts_bucket" {
```

Comment:
Really small thing but could we make the stack mgmt rather than mng
Also do we have documented how we plan to use the two buckets for each domain